exporters/postgresql/postgresql_exporter.go

```diff
@@ -66,10 +75,30 @@ func (exp *postgresqlExporter) Init(cfg plugins.PluginConfig, logger *logrus.Log
 	}
 	exp.db = db
 	<-ready
-	if rnd, err := exp.db.GetNextRoundToAccount(); err == nil {
+	rnd, err := exp.db.GetNextRoundToAccount()
+	if err == nil {
 		exp.round = rnd
+	} else if err == idb.ErrorNotInitialized {
+		exp.round = 0
+	} else {
+		return fmt.Errorf("Init() err getting next round: %v", err)
+	}
+	exp.ctx, exp.cf = context.WithCancel(context.Background())
```

Comment:
hopefully we should have a context passed into this function soon. Moving this to the top of the Init function should make it easier to swap out the `context.Background` later.

Reply:
+1